TriggerHandler.lua

```diff
@@ -771,7 +773,7 @@ function TriggerHandler:onActivateObject(superFunc,vehicle)
 			--seperarte for only loading Fuel in to motors!
 			if triggerHandler:isAllowedToLoadFuel() and fillableObject == vehicle then 
 				for fillTypeIndex, fillLevel in pairs(fillLevels) do
-					if fillTypeIndex == FillType.DIESEL  then 
+					if fillTypeIndex == FillType.DIESEL or fillTypeIndex == FillType.ELECTRICCHARGE then 
```

Comment:
and this probably, as isFuel(fillTypeIndex)? (Wondering if Giants did not do that already, have to check if they released the new code)

Reply:
havn't seen that function yet, but we probably could use this one.